counterparty-core/counterpartycore/lib/messages/fairminter.py

```diff
@@ -275,9 +284,9 @@ def unpack(message, return_dict=False):
             lock_description,
             lock_quantity,
             divisible,
-        ) = data_content[0:15]
+        ) = data_content[0:16]
```

Comment:
For here and the change below we could use `len - 1 ` to calculate this value instead of needing to update the number every time we add something